src/Social/Listener/Users.php

```diff
@@ -70,6 +71,9 @@ public function reply(Event $event, ModelInterface $entity): void
      */
     public function follow(Event $event, ModelInterface $entity): void
     {
-        Interactions::add(Di::getDefault()->get('userData'), $entity, InteractionsModel::FOLLOWING);
+        $user = Di::getDefault()->get('userData');
```

Comment:
can we use getting the userData from baka directly ?
https://github.com/bakaphp/baka/blob/0.6/src/Auth/UserProvider.php#L32